src/css/_layout.scss

```diff
@@ -1,35 +1,73 @@
-// Some frontend developers prefer to keep layout styles in a partial, like this.
-
-.content-container {
-	margin: 0 auto;
-	max-width: 900px;
-	padding: 1em;
-
-	@media screen and (min-width: 768px) {
-		display: grid;
-		grid-gap: 3em;
-		grid-template-columns: repeat(3, 1fr);
-		padding: 2em 1em 5em;
-	}
+body {
+    background-color: black;
+    margin: 7%
 }
 
-.page-heading {
-	grid-column: span 3;
+.lob-intro {
+    margin-bottom: 5in;
```

Comment:
Inches is a bit of an unusual unit to use here - nothing wrong with it specifically, but I'm curious why you used it. Is there a reason why you went with inches here vs. a percentage or screen-based unit like px or rem?